### PR TITLE
Add links to Transpile section

### LINKED ISSUE #439: Update "next steps" sections of transpile pages
At the moment in our transpile pages when we include a "next steps" section we only point to other documentation pages. We should update them to also include links to some learning platform content where appropriate.

For example, these would be good tutorials to link to:
- https://learning.quantum.ibm.com/tutorial/submit-transpiled-circuits
- https://learning.quantum.ibm.com/tutorial/reduce-transpiled-circuit-depth-with-circuit-cutting

### PR DESCRIPTION
**Pull request by beckykd**
Closes #439

### CODE REVIEW

**Comment by Eric-Arellano** [comment]
FYI this has a merge conflict

**Review by Eric-Arellano — APPROVED** [review]
LGTM! Not sure if you need Abby M or Abby C to sign off on the specific links.

File: docs/transpile/common-parameters.mdx [hunk 1]
```diff
@@ -100,7 +100,9 @@ These options influence how the transpiler works and are used to try and get bet
 ## Next steps
 
 <Admonition type="tip" title="Recommendation">
-    - [Default options and configuration settings](defaults-and-configuration-options)
-    - [Set the optimization level](set-optimization)
-    - [Submit transpiled circuits](https://learning.quantum.ibm.com/tutorial/submit-transpiled-circuits) with Qiskit Runtime on IBM Quantum Learning
+    - Review the [Default options and configuration settings](defaults-and-configuration-options) topic.
+    - Learn how to [Set the optimization level.](set-optimization)
+    - Try the [Submit transpiled circuits](https://learning.quantum.ibm.com/tutorial/submit-transpiled-circuits) tutorial.
+    - Review the [Transpile API documentation.](https://docs.quantum-computing.ibm.com/api/qiskit/transpiler)
```
Comment (abbycross) [inline docs/transpile/common-parameters.mdx:106]:
```suggestion
    - Review the [Transpile API documentation.](/api/qiskit/transpiler)
```

File: docs/transpile/defaults-and-configuration-options.mdx [hunk 2]
```diff
@@ -206,10 +206,12 @@ qiskit.compiler.transpile(unitary_synthesis_method='default', translation_method
...
-    - For examples, see [Representing quantum computers](representing_quantum_computers)
-    - [Transpile API](/api/qiskit/transpiler)
+    - Learn how to [Set the optimization level](set-optimization).
+    - Review more [Commonly used parameters](common-parameters).
+    - Learn how to [Set the optimization level when using Qiskit Runtime.](../run/advanced-runtime-options)
+    - Visit the [Transpile with pass managers](transpile-with-pass-managers) topic.
+    - For examples, see [Representing quantum computers.](representing_quantum_computers)
+    - Try the [Submit transpiled circuits](https://learning.quantum.ibm.com/tutorial/submit-transpiled-circuits) tutorial.
+    - Use transpilation with Qiskit Patterns in the [Reduce transpiled circuit depth, with circuit cutting](https://learning.quantum.ibm.com/tutorial/reduce-transpiled-circuit-depth-with-circuit-cutting) tutorial.
+    - Review the [Transpile API documentation](https://docs.quantum-computing.ibm.com/api/qiskit/transpiler).
```
Comment (abbycross) [inline docs/transpile/defaults-and-configuration-options.mdx:216]:
```suggestion
    - Review the [Transpile API documentation](/api/qiskit/transpiler).
```

File: docs/transpile/index.mdx [hunk 3]
```diff
@@ -45,5 +45,13 @@ For an overview of transpiling using pass managers, see [Transpile with pass man
...
\ No newline at end of file
+## Next steps
+
+<Admonition type="tip" title="Recommendations">
+    - To learn how to use the `transpile` function, start with the [Transpilation default settings and configuration options](defaults-and-configuration-options) topic.
+    - Continue learning about transpilation with the [Transpiler stages](transpiler-stages) topic.
+    - Try the [Submit transpiled circuits](https://learning.quantum.ibm.com/tutorial/submit-transpiled-circuits) tutorial.
+    - Use transpilation with Qiskit Patterns in the  [Reduce transpiled circuit depth, with circuit cutting](https://learning.quantum.ibm.com/tutorial/reduce-transpiled-circuit-depth-with-circuit-cutting) tutorial.
+    - Try an end-to-end example that uses transpiled circuits in the [Variational quantum eigensolver (VQE)](https://learning.quantum.ibm.com/tutorial/variational-quantum-eigensolver) tutorial.
+    - See the [Transpile API documentation.](https://docs.quantum-computing.ibm.com/api/qiskit/transpiler)
```
Comment (abbycross) [inline docs/transpile/index.mdx:56]:
```suggestion
    - See the [Transpile API documentation.](/api/qiskit/transpiler)
```

File: docs/transpile/representing_quantum_computers.mdx [hunk 4]
```diff
@@ -159,3 +159,11 @@ qc_t_cm_bg_et.draw('mpl', style='iqp', fold=-1, idle_wires=False)
 
 Note that by including the error information, the `VF2PostLayout` pass tries to find the optimal qubits to use, resulting in the same circuit that we found originally with the same physical qubits.
 
+## Next steps
+
+<Admonition type="tip" title="Recommendations">
+    - Understand [Transpilation default settings and configuration options.](defaults-and-configuration-options)
+    - Review the [Commonly used parameters for transpilation](common-parameters) topic. 
+    - Try the [Submit transpiled circuits](https://learning.quantum.ibm.com/tutorial/submit-transpiled-circuits) tutorial.
+    - See the [Transpile API documentation.](https://docs.quantum-computing.ibm.com/api/qiskit/transpiler)
```
Comment (abbycross) [inline docs/transpile/representing_quantum_computers.mdx:168]:
```suggestion
    - See the [Transpile API documentation.](/api/qiskit/transpiler)
```

File: docs/transpile/set-optimization.mdx [hunk 5]
```diff
@@ -151,5 +151,17 @@ Adjusting the optimization level can change other aspects of the circuit too, no
...
 </Admonition>
+
+## Next steps
+
+<Admonition type="tip" title="Recommendations">
+    - To learn how to use the `transpile` function, start with the [Transpilation default settings and configuration options](defaults-and-configuration-options) topic.
+    - Continue learning about transpilation with the [Transpiler stages](transpiler-stages) topic.
+    - Try the [Submit transpiled circuits](https://learning.quantum.ibm.com/tutorial/submit-transpiled-circuits) tutorial.
+    - Try the [Build repetition codes](https://learning.quantum.ibm.com/tutorial/build-repetition-codes) tutorial.
+    - See the [Transpile API documentation.](https://docs.quantum-computing.ibm.com/api/qiskit/transpiler)
```
Comment (abbycross) [inline docs/transpile/set-optimization.mdx:166]:
```suggestion
    - See the [Transpile API documentation.](/api/qiskit/transpiler)
```

**Review by abbycross — APPROVED** [review]
added comments to fix api urls so that they are internal links -- all else looks good to me!

**Pull request merged**